### ISSUE TITLE
Issue with per-game VMU setting/issue (only confirmed on Switch)

### ISSUE THREAD

**Issue by Shaneus2k2** [libretro/flycast#693]
If it's okay, I'll just copy/paste what I wrote in the gbatemp forum re: this core:
Pretty sure it's not the creation/saving of the game-specific VMU, but loading it again on startup. Can see individual game ones fine but it doesn't seem to load them when the core/game loads. No idea if it happens on other platforms either, but there's no mention of it on git so maybe it is only Switch :/ 

Silent_Gunner took additional steps to confirm the issue:
So, here's what I did: I loaded up MVC2, opened up the options in the Quick Menu, set the per-game VMU option to only the first slot, did a manual save, exited using Quit RA, and loaded up NX-Shell. The file for MVC2 for DC is in the root of the savefile folder. So, I decided to load up MVC2 again, and it acts like the savefile I created (for only the A1 slot, mind you) is a fresh new file, nothing unlocked by the prompt to create a new file for auto-saving purposes. After quitting RetroArch with the Quit RA option within RA, and restarting the game, it does indeed turn out that it doesn't load per-game saves for some reason. Shaneus' theory seems to be on point about per-game saves at this point in time!

So basically, it looks like it's creating per-game saves fine, but the core doesn't load them for whatever reason.

**Comment by flyinghead** [comment]
I can't reproduce it on linux so it might be a Switch-only issue.

**Comment by Shaneus2k2** [comment]
I can confirm it doesn't happen with Windows either, so I guess it's Switch-only too. Let me know if I need to close this here or ping whoever worked on porting the core (I think it was m4xw).

**Comment by flyinghead** [comment]
You can leave it open. And I'll ping @m4xw

**Comment by josereyesjrz** [comment]
Confirming it's a Switch issue #689
